### PR TITLE
User

### PR DESCRIPTION
**Pull request by msmithfc300**

### CODE REVIEW

File: src/Controller/LoginController.php
Comment (Phalenopsis) [inline src/Controller/LoginController.php:1]:
```php
$errors['password'] = 'Mot de passe incorrect';
// this is still not displaing correctly
```
i will look as soon as we will could pull this, with all branch merge

**Review by Phalenopsis — APPROVED** [review]
Nice. Good job Martha

File: src/Controller/LoginController.php
Comment (msmithfc300) [reply src/Controller/LoginController.php:1]: Thank you! I added the session_start() today and the homepage now displays "you are logged in as Martha" when i test it :) so the one remaining bug is this password error. I also have to test the logout method!

**Comment by yoyodes1000** [comment]
nickel

**Review by yoyodes1000 — APPROVED** [review]

**Pull request merged**